shoop/apps/provides.py

```diff
@@ -178,10 +178,44 @@ def load_module(setting_name, provide_category):
     :return: An object.
     :rtype: object
     """
+    setting_value = _get_settings_value(setting_name)
+    return _load_module(provide_category, setting_name, setting_value)
+
+
+def load_modules(setting_name, provide_category):
+    """
+    Load a list of modules from a module setting.
+
+    The value of the setting must be a list of module identifiers for the given provide category.
+
+    The modules are returned in the same order they are declared in the settings.
+
+    :param setting_name: The setting name for the identifier list
+    :type setting_name: str
+    :param provide_category: The provide category for the identifier lookup (e.g. `tax_module`)
+    :type provide_category: str
+    :return: A list of objects
+    :rtype: list[object]
+    """
+    setting_value = _get_settings_value(setting_name)
+
+    if not isinstance(setting_value, list):
+        raise ImproperlyConfigured("The setting `%s` MUST be a `list`." % setting_name)
```

Comment:
Drop this check it's not needed and e.g. tuple is a fine value too. If the setting value is not iterable it will error out later for sure, which is as good as this, but less demanding and less code.